File: 백준/1371.py
import sys

sys.stdin = open('input.txt')
words = sys.stdin.read()
words = words.replace('\n',"")
words = words.replace(" ",'')

dict = {}

for i in words:
    for j in i:
        if j in dict:
            dict[j] += 1
        else:
            dict[j] = 1
word = []
for k, v in dict.items():
    if max(dict.values()) == v:
        word.append(k)

word.sort()
for l in word:
    print(l,end='')

File: JavaScript/백준/Bronze 2/1371.js
const fs = require('fs')
let input = fs.readFileSync('./1371.txt').toString().replace(/\n|\r/g, "").split(" ")

let wordObject = Object()
let maxValue = 0
for (words of input) {
    for (word of words) {
        if (wordObject[word]) {
            wordObject[word] += 1
        } else {
            wordObject[word] = 1
        }
        if (wordObject[word] > maxValue) {
            maxValue = wordObject[word]
        }
    }
}

const wordArray = []

for (const [key, value] of Object.entries(wordObject)) {
    if (value == maxValue) {
        wordArray.push(key)
    }
}

const answer = wordArray.sort()
console.log(answer.join(""))